File: bsw112/2021-01-28/핸드폰번호가리기.java
public class 핸드폰번호가리기 {
    public String solution(String phone_number) {
        //charArr로 바꿔도 될듯
        StringBuilder answer = new StringBuilder(phone_number);
        for (int i = 0; i < answer.length() - 4; i++) {
            answer.replace(i, i + 1, "*");
        }
        return answer.toString();
    }

}

File: giwan/핸드폰번호가리기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string phone_number) {
    string answer = "";
    int i;
    for(i=0;i<phone_number.length()-4;i++){
        answer+="*";
    }
    for(;i<phone_number.length();i++){
        answer+=phone_number[i];
    }
    return answer;
}